package.json

```diff
@@ -8,15 +8,14 @@
   ],
   "dependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@1.2.2",
-    "@edx/frontend-component-footer": "12.2.0",
-    "@edx/frontend-enterprise-catalog-search": "^5.6.0",
-    "@edx/frontend-enterprise-hotjar": "2.0.0",
-    "@edx/frontend-enterprise-logistration": "^4.1.0",
-    "@edx/frontend-enterprise-utils": "^4.1.0",
-    "@edx/frontend-platform": "5.0.0",
-    "@edx/paragon": "20.45.4",
+    "@edx/frontend-component-footer": "13.0.2",
+    "@edx/frontend-enterprise-catalog-search": "7.0.0",
+    "@edx/frontend-enterprise-hotjar": "4.0.0",
+    "@edx/frontend-enterprise-logistration": "6.0.0",
+    "@edx/frontend-enterprise-utils": "6.0.0",
+    "@edx/frontend-platform": "7.1.0",
     "@loadable/component": "5.16.3",
-    "@openedx/paragon": "^22.1.1",
+    "@openedx/paragon": "^21.5.7",
```

Comment:
pinning the dependency here throws an error because `@edx/frontend-component-footer` requires an older version
 
```
npm ERR! Could not resolve dependency:
npm ERR! peer @openedx/paragon@">= 21.11.3 < 23.0.0" from @edx/frontend-component-footer@13.0.2
npm ERR! node_modules/@edx/frontend-component-footer
npm ERR!   @edx/frontend-component-footer@"13.0.2" from the root project
```